frontend/src/components/Nav.jsx

```
@@ -1,69 +1,78 @@
+                    </NavLink>
+                  ))}
                 </div>
-            </div>
-            <hr />
-        </div>
-    );
+              </NavigationMenuContent>
+            </NavigationMenuItem>
+            <NavigationMenuItem>
+              <NavigationMenuTrigger>
+                <i className="nf nf-md-account_circle"></i>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)